gravitee-apim-portal-webui/src/app/pages/catalog/filtered-catalog/filtered-catalog.component.ts

```diff
@@ -155,7 +155,7 @@ export class FilteredCatalogComponent implements OnInit {
   }
 
   _load() {
-    if (this.page === 1 && this.hasPromotedApiMode()) {
+    if (this.page === 1) {
```

Comment:
Should we keep the boolean hasPromotedApiMode test, to avoid calling the API to retrieve promoted API, if parameter is disabled ?
It seems actually, the promoted API is retrieved event if not displayed.

Reply:
@marcambier done